scripts/check-test-performance.cjs

```
@@ -0,0 +1,409 @@
+    
+    // Extract summary if available
+    const summaryMatch = result.match(/Test Files.*?(\d+).*?Tests.*?(\d+).*?Time.*?(\d+(?:\.\d+)?)(m?s)/);
+    if (summaryMatch) {
+      testCount = parseInt(summaryMatch[2]);
+      let timeValue = parseFloat(summaryMatch[3]);
+      if (summaryMatch[4] === 'ms') {
+        // Already in milliseconds
+      } else if (summaryMatch[4] === 's') {
+        timeValue *= 1000; // Convert to ms
+      }
+      totalRuntime = Math.min(totalRuntime, timeValue); // Use more accurate time if available
```

Biome (2.1.2)

[error] Can't assign totalRuntime because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)